src/librbd/api/Group.cc

```diff
@@ -1226,24 +1216,118 @@ int Group<I>::snap_rename(librados::IoCtx& group_ioctx, const char *group_name,
   return 0;
 }
 
+int convert_cls_rbd_GroupSnapshot_to_librbd_group_snap_info2_t(
+    librados::IoCtx& group_ioctx, const std::string& group_id,
+    const cls::rbd::GroupSnapshot* cls_group_snap,
```

Comment:
Observation-only, never-NULL parameters should be passed for const reference:
```suggestion
    const cls::rbd::GroupSnapshot& cls_group_snap,
```